### PR TITLE
Incorrect and incomplete translations in Turkish have been corrected.

### PR DESCRIPTION
**Pull request by AbdulkadirAvan**
I hope I didn't do anything wrong.

### CODE REVIEW

**Comment by lonnieezell** [comment]
@obozdag do you concur?

File: Language/tr/Validation.php [hunk 1]
```diff
@@ -27,25 +27,25 @@
...
 	'integer'               => '{field} alanı bir tamsayı içermeli.',
 	'is_natural'            => '{field} alanı yalnız rakam içermeli.',
 	'is_natural_no_zero'    => '{field} alanı yalnız rakam içermeli ve sıfırdan büyük olmalı.',
-   'is_not_unique'         => 'The {field} field must contain a previously existing value in the database.',
+	'is_not_unique'         => '{field} alanı, veritabanında önceden var olan bir değeri içermeli.',
 	'is_unique'             => '{field} alanı eşsiz bir değer içermeli.',
 	'less_than'             => '{field} alanı {param} den küçük bir sayı içermeli.',
 	'less_than_equal_to'    => '{field} alanı {param} den küçük veya eşit bir sayı içermeli.',
-	'matches'               => '{field} alanı {param} ile aynı olmamalı.',
+	'matches'               => '{field} alanı {param} ile aynı olmalı.',
```
Comment (obozdag) [inline Language/tr/Validation.php:45]: '{field} alanı ile {param} alanı aynı olmalı.'

**Comment by obozdag** [comment]
Missing translations are completed and the incorrect translation is corrected. Thanks.

**Comment by obozdag** [comment]
@lonnieezell Yes I do.

**Pull request merged**